config/initializers/content_security_policy.rb

```diff
@@ -25,8 +25,9 @@
 
   policy.script_src(
     :self,
-    "https://www.googletagmanager.com/",
-    "https://cdn.segment.com/"
+    'https://www.googletagmanager.com/',
+    'https://cdn.segment.com/',
+    'https://cdn.emojicom.io/'
```

Comment:
Should we standardise on linter config?

Reply:
I prefer double quotes so that they're consistent everywhere. I assumed it was a project-based linter setting, but perhaps it was something in my local setup?

Do you want me to change them back here (or rather remove that commit) or we'll fix it moving forward?

Reply:
Looks like there's nothing specific to quotations in the our RuboCop/linter config. If that's the convention you've seen kicking around then I'd stick with what you have.